pages/search.js

```diff
@@ -252,12 +249,12 @@ export default function Search({
               ref={searchInputRef}
               placeholder="Search user by name or tags; eg: open source, reactjs or places; eg: London, New York"
               name="keyword"
-              defaultValue={userSearchParam}
+              defaultValue={searchTerm}
             />
           </form>
```

Comment:
```suggestion
          <Button className={"my-2"}>Search</Button>
          </form>
```

Reply:
![Screenshot (49)](https://github.com/EddieHubCommunity/BioDrop/assets/73095526/bc705df6-15c2-451a-8738-5b06267ff870)

added the search button. not designed a good-looking search bar because this PR is more focused on search functionality. search bar re-design could come in another PR as I don't want to add more complexity to this PR, Thanks